adapter/serviceControl/client.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package serviceControl
```

Comment:
please call this package `servicecontrol` and put it in a directory that is all lowercase as well.

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.
-- https://blog.golang.org/package-names